### DISCUSSION TITLE
Is there a sql client you can use to run queries on s3 deltalake tables?

### DISCUSSION THREAD

**Discussion by jensqin** [delta-io/delta-rs#2015] [Q&A]
I am aware that we can run queries using duckdb engine, and I wonder if there is a sql client/UI for s3 deltalake tables.

**Comment by yefetBenTili** [comment]
Something you could try is to create a glue table from the S3 Delta Lake then use any SQL client and jdbc connect to Athena and query the table

**Reply by ahirner** [reply]
You may try https://github.com/datafusion-contrib/datafusion-dft @jensqin
